bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.airq.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link airqConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Aurelio Caliaro - Initial contribution
+ */
+
+@NonNullByDefault
+public class airqConfiguration {
+    public @NonNullByDefault({}) String ipaddress;
+    public @NonNullByDefault({}) String password;
```

Comment:
```suggestion
    public String ipAddress="";
    public String password="";
```

Since you have it as ipAddress in the XML it needs to be identical here for the suggestion I made with "config" to work.